### DISCUSSION TITLE
Is it possible to use podman within ubuntu via podman desktop?

### DISCUSSION THREAD

**Discussion by sethwenzel** [podman-desktop/podman-desktop#2472] [Q&A]
Podman desktop seems to want to set up a new WSL distro (like docker does) BUT, can I use my podman _within_ WSL instead?  I am a bit of a linux noob...

My setup:
Windows 11 with WSL2
Ubuntu 22.04 LTS
Podman installed in Ubuntu (3.4.4)
Podman desktop installed on windows: Podman Desktop v0.15.0

I assume there's some sort of TCP setup I need to do with Podman on Ubuntu, I just can't find good docs about it.  Is there any way to do this, or should I just continue using Podman cli in Ubuntu and not worry about Podman Desktop?

**Comment by benoitf** [comment]
Hello @sethwenzel,

Podman Desktop connects to a socket(npipe) on the host (your Windows system)
This local npipe is automatically setup by the `podman machine init|start` command

if you want to do it manually, you could follow this thread: https://github.com/containers/podman/discussions/16130#discussioncomment-3945343
